sdk/data/aztables/client_test.go

```diff
@@ -216,7 +217,7 @@ func TestInsertEntity(t *testing.T) {
 			defer delete()
 
 			// 1. Create Basic Entity
-			entityToCreate := createSimpleEntity(1, "partition")
+			entityToCreate := createSimpleEntityWithRowKey(1, "parti'tion", "one'")
```

Comment:
Lets beef up the test suite around  this.  I know the current fix makes it moot, but if that impl ever changed it would be good to know we have a strong escaping suite.

Some ideas:
Single quote at beginning of key/values
Single quote at end of key/values
Multiple single quotes in a row
Single quote only

Reply:
Added tests (thanks for reminding me).